express-api/src/services/projects/projectsServices.ts

```diff
@@ -556,8 +556,8 @@ const handleProjectAgencyResponses = async (
         ...resp,
         ProjectId: newProject.Id,
         CreatedById: newProject.CreatedById,
-        DeletedById: null,
-        DeletedOn: null,
+        DeletedById: newProject.DeletedById,
+        DeletedOn: newProject.DeletedOn,
```

Comment:
This may change how the agency responses are handled by overwriting deleted responses' "Deleted" fields. Will test.

Reply:
Had to change this back. If we don't reset these to null, then the agency's interest record cannot be added again once removed.